packages/components/src/components/domainproperties/samples/SampleTypeDesigner.tsx

```diff
@@ -57,6 +57,13 @@ interface Props {
     includeDataClasses?: boolean
     headerText?: string
     helpTopic?: string
+    useSeparateDataClasses?: boolean
```

Comment:
```suggestion
    useSeparateDataClassesAliasMenu?: boolean
```
Longer name, but more descriptive.